app/Http/Transformers/ActionlogsTransformer.php

```diff
@@ -143,39 +156,49 @@ public function transformCheckedoutActionlog (Collection $accessories_users, $to
      * @param  array $clean_meta
      * @return array
      */
-    public function changedInfo(array $clean_meta)
-    {
+    public function changedInfo(array $clean_meta) {
+
+        $location = Location::withTrashed()->get();
+        $company = Company::withTrashed()->get();
```

Comment:
This fails for me since `Company` doesn't have the `SoftDeletes` trait on it.

Reply:
Same (and good catch)

Reply:
@Godmartinz I think the lines starting at 168 in [this branch](https://github.com/snipe/snipe-it/compare/master...marcusmoore:snipe-it:bug/sc-23635#diff-7fe056d76c09808dac923c4639161d587c3fff281a01122f3e10c4a781674a65R168-R175) should handle this for companies.

Reply:
added the fix